src/Microsoft.ML.Data/Transforms/Hashing.cs

```diff
@@ -1451,6 +1451,11 @@ private bool SaveAsOnnxCore(OnnxContext ctx, int iinfo, string srcVariable, stri
             void ISaveAsOnnx.SaveAsOnnx(OnnxContext ctx)
             {
                 Host.CheckValue(ctx, nameof(ctx));
+
+                const int minimumOpSetVersion = 11;
```

Comment:
>11 [](start = 48, length = 2)

Why is opset 11 required here?

Reply:
As some or one Op(BitShift) used in Hashing's onnx conversion requires version 11.